doc/user/content/self-managed-deployments/installation/install-on-aws.md

```diff
@@ -0,0 +1,337 @@
+| IAM Role | IRSA role for Kubernetes service account access |
+
+### Kubernetes Add-ons
+
+| Resource | Description |
+|----------|-------------|
+| AWS Load Balancer Controller | For managing Network Load Balancers |
+| cert-manager | Certificate management controller for Kubernetes that automates TLS certificate provisioning and renewal |
+| Self-signed ClusterIssuer | Provides self-signed TLS certificates for Materialize instance internal communication (balancerd, console). Used by the Materialize instance for secure inter-component communication. |
+
+### Materialize
+
+| Resource | Description |
+|----------|-------------|
+| Operator | Materialize Kubernetes operator in the `materialize` namespace |
+| Instance | Single Materialize instance in the `materialize-environment` namespace |
+| Network Load Balancer | Dedicated NLB for access to Materialize {{< yaml-table data="self_managed/default_ports" >}} |
+
+
+## Prerequisites
+
+### AWS Account Requirements
+
+An active AWS account with appropriate permissions to create:
+- EKS clusters
+- RDS instances
+- S3 buckets
+- VPCs and networking resources
+- IAM roles and policies
+
+### Required Tools
+
+- [Terraform](https://developer.hashicorp.com/terraform/install?product_intent=terraform)
+- [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html)
+- [kubectl](https://docs.aws.amazon.com/eks/latest/userguide/install-kubectl.html)
+- [Helm 3.2.0+](https://helm.sh/docs/intro/install/)
+
+
+### License Key
+
+{{< yaml-table data="self_managed/license_key" >}}
+
+## Getting started: Simple example
+
+{{< warning >}}
+
+{{< self-managed/terraform-disclaimer >}}
+
+{{< /warning >}}
+
+### Step 1: Set Up the Environment
+
+1. Open a terminal window.
+
+1. Clone the Materialize Terraform repository and go to the
+   `aws/examples/simple` directory.
+
+   ```bash
+   git clone https://github.com/MaterializeInc/materialize-terraform-self-managed.git
+   cd materialize-terraform-self-managed/aws/examples/simple
+   ```
+
+1. Ensure your AWS CLI is configured with the appropriate profile, substitute
+   `<your-aws-profile>` with the profile to use:
+
+   ```bash
+   # Set your AWS profile for the session
+   export AWS_PROFILE=<your-aws-profile>
+   ```
+
+### Step 2: Configure Terraform Variables
+
+1. Create a `terraform.tfvars` file with the following variables:
+
+   - `name_prefix`: Prefix for all resource names (e.g., `simple-demo`)
+   - `aws_region`: AWS region for deployment (e.g., `us-east-1`)
+   - `aws_profile`: AWS CLI profile to use
+   - `license_key`: Materialize license key
+   - `tags`: Map of tags to apply to resources
+
+   ```hcl
+   name_prefix = "simple-demo"
+   aws_region  = "us-east-1"
+   aws_profile = "your-aws-profile"
+   license_key = "your-materialize-license-key"
+   tags = {
+     environment = "demo"
+   }
+   # internal_load_balancer = false   # default = true (internal load balancer). You can set to false = public load balancer.
```

Comment:
Almost ... waiting on resolution of https://github.com/MaterializeInc/materialize-terraform-self-managed/issues/129